CHEATSHEET.md

```diff
@@ -14,88 +12,97 @@ import claripy #the solver engine
 ```
 
 Loading the binary
+```python
+proj = angr.Project("/path/to/binary", auto_load_libs=False) # auto_load_libs False for improved performance
+```
+
+## States
+
+Create a SimState object
 
 ```python
-proj = angr.Project("/path/to/binary", load_options={'auto_load_libs': False} ) # auto_load_libs False for improved performance
+state = proj.factory.entry_state()
 ```
 
-## Path Groups
+## Simulation Managers
 
-Generate a path group object
+Generate a simulation manager object
 
 ```python
-path_group = proj.factory.path_group(state, threads=4)
+simgr = proj.factory.simulation_manager(state)
 ```
 
-## Exploring and analysing pathgroups
+## Exploring and analysing states
 
 Choosing a different Exploring strategy
 
 ```python
-path_group.use_technique(angr.exploration_techniques.DFS())
+simgr.use_technique(angr.exploration_techniques.DFS())
 ```
-
-Explore Pathgroup until one pathgroup at one of the adresses from `find=` is found
+Symbolically execute until we find a state satisfying our `find=` and `avoid=` parameters
 
 ```python
 avoid_addr = [0x400c06, 0x400bc7]
 find_addr = 0x400c10d
-path_group.explore(find=find_addr, avoid=avoid_addr)
+simgr.explore(find=find_addr, avoid=avoid_addr)
 ```
 
 ```python
-found = path_group.found[] # The list of paths that reached find condition from explore
-found.state.se.any_str(sym_arg) # Return a concrete string value for the sym arg to reach this state
+found = simgr.found[0] # A state that reached the find condition from explore
+found.solver.eval(sym_arg, cast_to=str) # Return a concrete string value for the sym arg to reach this state
 ```
 
-Explore pathgroup until lambda is `True`
+Symbolically execute until lambda expression is `True`
 
 ```python
-path_group.step(until=lambda p: p.active[0].addr >= first_jmp)
+simgr.step(until=lambda sm: sm.active[0].addr >= first_jmp)
 ```
 
 This is especially useful with the ability to access the current STDOUT or STDERR (1 here is the File Descriptor for STDOUT)
 
 ```python
-path_group.explore(find=lambda p: "correct" in p.state.posix.dumps(1))
+simgr.explore(find=lambda sm: "correct" in sm.state.posix.dumps(1))
```

Comment:
This needs an additional change - the argument to the lambda isn't a simulation manager, it's a state. It used to be a path, which is why there's `.state`, so that should be removed too.